src/drivers/src/motors_def_cf2.c

```diff
@@ -301,6 +301,31 @@ static const MotorPerifDef BOLT_M4_BL =
     .preloadConfig = TIM_OC4PreloadConfig,
 };
 
+// Bolt 1.1 M4, PB10, TIM2_CH3, Brushless config
```

Comment:
Hi is there only here a addition for motor M4?

Reply:
Yes, this is the only HW change.